site/content/en/reference/live/apply/_index.md

```diff
@@ -91,6 +91,47 @@ is added to the name. Example:
 test-grouping-object-17b4dba8
 ```
 
+### Ordering
+
+`kpt live apply` will sort the resources before applying them. This makes sure
+namespaces are applied before resources going into the namespace, configmaps
+are applied before Deployments and StatefulSets that might reference them, and
+other known dependencies between the builtin kubernetes resource types.
+Custom ordering of resources is not supported.
+
+During pruning, the same rules are used, but resources will be deleted in
+reverse order. Note that this does not wait for a resource to be deleted
+before continuing to delete the remaining resources.
```

Comment:
what's a concrete scenario where it's useful to have non-blocking deletion order?

Reply:
Yeah, this helps in some situations, but it doesn't help when there are finalizers to delay deletion of resources. We do want to support that too, but it requires bigger changes.